packages/manager/modules/telecom-dashboard/src/telecom-dashboard.scss

```diff
@@ -0,0 +1,4 @@
+@import "~bootstrap4/scss/_functions";
```

Comment:
Shouldn't be scoped? (eg: https://github.com/ovh-ux/manager/pull/88/files#diff-654a8db825ac61ba415b4cd2ab24f232R1)